### PR TITLE
Fix image coding in cloud soar

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) fixes the image coding and does some other cleanup in the Cloud SOAR articles:
https://help.sumologic.com/docs/cloud-soar/

Issue number: Asana ticket "[Clean up links and image coding in SIEM and SOAR](https://app.asana.com/0/1183741177809740/1204726032012276/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cloud-soar/automation.md [hunk 1]
```diff
@@ -81,8 +81,6 @@ Watch this micro lesson to learn how to create custom playbooks.
 
 To configure a new Playbook, click the cog icon (<img src={useBaseUrl('img/cloud-soar/cog.png')} alt="cog menu" width="20"/>) > **Automation**.
 
-![Playbook](/img/cloud-soar/image71.png)
```
Comment (kimsauce) [inline docs/cloud-soar/automation.md:84]: @jpipkin1 this img was deleted without replacement code - was this intentional?

**Review by kimsauce — COMMENTED** [review]
one Q

File: docs/cloud-soar/automation.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cloud-soar/automation.md:84]: Yep.

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**